aws-ecs/README.md

```diff
@@ -0,0 +1,59 @@
+
+## Exporting AWS ECS Stats to Wavefront using Telegraf
+
+Amazon ECS, input plugin uses the Amazon ECS metadata and stats v2 or [v3][task-metadata-endpoint-v3] API endpoints to gather stats on running containers in a Task.
+
+The wavefronthq/aws-ecs-wavefront container configured with telegraf must be run in the same Task as the workload it is inspecting to export stats to Wavefront (https://wavefront.com) output plugin.
```

Comment:
I think we can shorten this and just mention that this task uses telegraf ecs plugin and provide link to plugin.